### ISSUE TITLE
Formatting of annotations makes the really hard to access

### ISSUE THREAD

**Issue by dmj** [markupdeclaration/markupdeclaration.org#16]
The annotations are stuffed in a 50px height box with scroll bars. A user has to move the mouse over the annotation box to access the information therein. Maybe better use blockquote and a smaller font?

**Comment by ndw** [comment]
Yeah. I wasn't sure about that. Some of them are quite long, so making them initially small seemed ok. But I grant the accessibility issue. If you mouse over, or tab through them, they fully expand for you, don't they?

**Comment by dmj** [comment]
They do expand. I have the feeling that this is also an editorial issue: By nature the _annotated_ bibliography packs a lot of information. I think some kind of editorial control needs to be applied: Maybe every entry gets a succinct summary that is shown by default, the full annotations can be viewed on request?

**Comment by LinguaCelta** [comment]
Maybe the thing to do is to have short summaries of a set length (say, no more than a sentence per entry), and then a link to each entry individually, where the full annotated text can be read? Would that be silly amounts of work?

**Comment by btusdin** [comment]
> On Sep 24, 2018, at 9:51 AM, Bethan Tovey <notifications@github.com> wrote:
> 
> Maybe the thing to do is to have short summaries of a set length (say, no more than a sentence per entry), and then a link to each entry individually, where the full annotated text can be read? Would that be silly amounts of work?
> 

It would certainly be a lot more work. I suggest we try it this way for a while, with the expanded initial window. We need to find the sweet spot between making it easy for people to contribute and making the document easy to read. In the beginning it may be that ease of contributing is even more important than ease of reading. 

— Tommie
